textBasedAdventure.js

```diff
@@ -1,5 +1,150 @@
-const readline1 = require('readline-sync')
+const readline1 = require("readline-sync");
 
-let nameInput = readline.question("Enter your name: ")
+let nameInput = readline1.question("Enter your name: ");
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+console.log(`Hello ${nameInput}! \nWelcome to Mystery Inc.`);
+game();
+
+
+function game() {
+  if (readline1.keyInYN("Would you like to play and decide your own fate?")) {
+    intro();
+  } else {
+    quitGame();
+  }
+}
+
+function quitGame() {
+  console.log("Sorry to hear, enjoy not knowing");
+  process.exit();
+}
+
+
+
+
+function intro(){
+  console.log(
+    "Life is a mystery and things are always changing.\nEvery ACTION has a REACTION so BE CAREFUL it could cost you your LIFE. \nIt's raining and dark outside, the fog clouds your vision as you open the front door. The hairs on the back of your neck begin to stand, something is wrong. What do you do?"
+  )
+  startGame()
+}
+
+
+function startGame() {
+  
+  let answer1 = ["A", "B", "C"];
+  while (answer1 !== null) {
```

Comment:
What do you want this while loop to do ? 
`answer1` is never going to `null` because you are defining it in line 35 and assigning it some value/data